cmd/forwarder/run/run.go

```diff
@@ -67,6 +69,9 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
 
 	var ep []forwarder.APIEndpoint
 
+	logger.Infof("Forwarder %s (%s)", version.Version, version.Commit)
+	logger.Debugf("resource limits: GOMAXPROCS=%d GOMEMLIMIT=%s", runtime.GOMAXPROCS(0), os.Getenv("GOMEMLIMIT"))
```

Comment:
nit: I don't like this empty log `GOMEMLIMIT=`, even if it's just a debug mode.

Reply:
Would you like it like `<unknown>`

Reply:
Yes, please.